SSA/Projects/LLVMRiscV/Pipeline/xor.lean

```diff
@@ -25,8 +25,9 @@ def xor_riscv: Com  LLVMPlusRiscV ⟨[.llvm (.bitvec 64), .llvm (.bitvec 64)]⟩
       llvm.return %2 : i64
   }]
 
-  def llvm_xor_lower_riscv: LLVMPeepholeRewriteRefine 64 [Ty.llvm (.bitvec 64), Ty.llvm (.bitvec 64)] :=
-  {lhs := llvm_xor, rhs := xor_riscv}
+def llvm_xor_lower_riscv: LLVMPeepholeRewriteRefine 64 [Ty.llvm (.bitvec 64), Ty.llvm (.bitvec 64)] where
+  lhs := llvm_xor
+  rhs := xor_riscv
```

Comment:
@luisacicolini if you're cleaning up the pipeline files, this is the syntax I'd suggest we use